packages/appstore/src/components/add-more-wallets/add-more-wallets.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Text, Loading } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { useAvailableWallets } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import CarouselContainer from './carousel-container';
+import AddWalletCard from './wallet-add-card';
+
+import './add-more-wallets.scss';
+
+const AddMoreWallets = observer(() => {
+    const { data, isLoading } = useAvailableWallets();
+    const { ui: is_mobile } = useStore();
+
+    if (!data) return <Loading is_fullscreen={false} />;
```

Comment:
```suggestion
```
This is not needed since you already add an isLoading check below:

Ref: https://tanstack.com/query/v4/docs/react/guides/queries

<img width="504" alt="Screenshot 2023-09-21 at 5 14 16 PM" src="https://github.com/binary-com/deriv-app/assets/75345074/4ab4ead4-c951-4cd3-af92-5bcd717c86fc">